package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "performance",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Performance plugin from the WordPress Performance Group, which is a collection of standalone performance modules.",
+  "author": "The WordPress Contributors",
+  "license": "GPL-2.0-or-later",
+  "keywords": [
+    "performance",
+    "wordpress"
+  ],
+  "homepage": "https://wordpress.org/plugins/performance-lab/",
+  "repository": "git+https://github.com/WordPress/performance.git",
+  "bugs": {
+    "url": "https://github.com/WordPress/performance/issues"
+  },
+  "engines": {
+    "node": ">=10.0.0",
+    "npm": ">=6.9.0 <7"
+  },
+  "devDependencies": {
+    "@wordpress/env": "^4.1"
+  },
+  "scripts": {
+    "format-php": "wp-env run composer run-script format",
```

Comment:
what is the advantage of running this in docker? can it just be `composer run-script format`

Reply:
Having things run in a controlled environment makes them more predictable and avoids issues like "it doesn't work on my system". Also you don't even need to have Composer installed locally by having them run in the container, which means one less dependency required. In other words, the `npm` approach allows to do things with the least setup.

However, any of these commands can also be run locally by calling the respective `composer` command directly (for PHPUnit tests that requires some extra setup though), so this should provide the flexibility for folks that prefer to not use Docker (which however requires a bit more work to get set up correctly).

Reply:
👍🏼  makes sense; I'm asking mostly because I don't typically use docker for my local environment and I'd rather run these commands more directly. Also, I suspect it will be faster, for example if I want to run phpunit tests repeatedly during development.

Reply:
Definitely, that's why allowing users to alternatively do that is crucial. 👍